pkgs/servers/nextcloud/default.nix

```diff
@@ -44,13 +44,7 @@ let
         changelog = "https://nextcloud.com/changelog/#${lib.replaceStrings [ "." ] [ "-" ] version}";
         description = "Sharing solution for files, calendars, contacts and more";
         homepage = "https://nextcloud.com";
-        maintainers = with lib.maintainers; [
-          schneefux
-          bachp
-          globin
```

Comment:
@globin @schneefux since I don't recall you doing NC maintenance in the past years(?), I didn't add you to the team. If you want to be part of regardless, let me know.